addons/pkg/util/cluster_util.go

```diff
@@ -68,7 +67,7 @@ func GetClusterByName(ctx context.Context, c client.Client, namespace, name stri
 }
 
 // GetClustersByTKR gets the clusters using this TKR
-func GetClustersByTKR(ctx context.Context, c client.Client, tkr *runtanzuv1alpha1.TanzuKubernetesRelease) ([]*clusterv1beta1.Cluster, error) {
+func GetClustersByTKR(ctx context.Context, c client.Client, tkr client.Object) ([]*clusterv1beta1.Cluster, error) {
```

Comment:
Should this be changed back since you have separate methods now?

Reply:
Thanks for this. Addressed in latest commit (had to squash before merge with main).